tests/test_bezier_extraction.py

```diff
@@ -45,7 +45,7 @@ def test_extraction(self):
     def test_extraction_matrice_bspline_3D(self):
         """Create matrices to extract splines"""
         # Init b-splines
-        bspline = c.splinepy.BSpline(**c.b3P3D)
+        bspline = self.spline_b3p3d()
```

Comment:
Why is this not called `bspline_3p3d`? This is just as confusing as it was before

Reply:
I use the abbreviations given in the common.py file. But of course I can rename it.